src/main/java/org/datadog/jenkins/plugins/datadog/publishers/DatadogJenkinsPublisher.java

```diff
@@ -26,15 +26,18 @@ of this software and associated documentation files (the "Software"), to deal
 package org.datadog.jenkins.plugins.datadog.publishers;
 
 import hudson.Extension;
+import hudson.PluginManager;
+import hudson.PluginWrapper;
 import hudson.model.PeriodicWork;
 import hudson.model.Project;
 import jenkins.model.Jenkins;
 import org.datadog.jenkins.plugins.datadog.DatadogClient;
 import org.datadog.jenkins.plugins.datadog.DatadogUtilities;
 import org.datadog.jenkins.plugins.datadog.clients.ClientFactory;
+import org.datadog.jenkins.plugins.datadog.model.PluginData;
```

Comment:
Seems like you forgot to commit this file?

Reply:
So I did! Committed now.